### ISSUE TITLE
Note Contents All On Single Line

### ISSUE THREAD

**Issue by jseel001** [jeffkowalski/geeknote#3]
Hello,

After a note is saved, the contents of the note are confined to a single line. This happens in both vim and nano.

Example:

Expected behaviour:

line_1
line_2
line_3

Behaviour:

line_1 line_2 line_3

**Comment by arielchuri** [comment]
Same error here.

**Comment by jaguarondi** [comment]
Comparing to the official repo, I found the problem comes from the default note extension which is ".org" while it is ".markdown" in the official branch. 

Here is the code that is using the note extension. I don't know what's broken afterwards if the extension is ".org".

       noteExtension = Storage().getUserprop('note_ext')
        if not noteExtension:
            noteExtension = config.DEF_NOTE_EXT
        (tempfileHandler, tempfileName) = tempfile.mkstemp(suffix = noteExtension)

So a way to go around the problem is to set the default note extension in settings to ".markdown":

```# geeknote settings --note_ext ".markdown"

Or edit config.py and change this line
    DEF_NOTE_EXT = ".markdown"

**Comment by jeffkowalski** [comment]
Fixed, per @jaguarondi "Note ext" #13
